cv_next/app/cv/[cvId]/components/commentSection/comment.tsx

```diff
@@ -28,69 +29,71 @@ export default function Comment({
   commentsOfComment = [],
   setCommentsOfComments,
 }: {
-  comment: CommentModel
-  userId: string
-  childDepth?: number
-  cv: CvModel
-  commentsOfComment: any[]
+  comment: CommentModel;
```

Comment:
This is a big prop list, please extract to interface (doesn't need to be exposed as public)